tests/test_claim_ten_test_gate.py

```
@@ -0,0 +1,132 @@
+    return (
+        "ТОО «СтройИнжиниринг KZ» и ТОО «Астана Девелопмент» заключили Договор подряда № 07/26 от 15.03.2026 "
+        "на монтажные работы стоимостью 8 400 000 тенге. 30.04.2026 подписан акт выполненных работ. "
+        "Оплата не произведена. Договором предусмотрена неустойка 0,1% за каждый календарный день просрочки. "
+        "Неустойка за заявленный период составляет 840 000 тенге. 10.06.2026 направлена досудебная претензия. "
+        "Обе организации зарегистрированы в городе Астана."
+    )
+
+
+def test_objective_contains_all_ten_tests_and_legal_cost_correction():
+    for index in range(1, 11):
+        assert f"Т{index}." in TEN_TEST_OBJECTIVE
```

Ruff (0.16.2)

[warning] String contains ambiguous `Т` (CYRILLIC CAPITAL LETTER TE). Did you mean `T` (LATIN CAPITAL LETTER T)?

(RUF001)